src/utils.js

```diff
@@ -52,7 +52,8 @@ function getDefaultConfig() {
 		repeat: false,				// This will repeat callback every n times duration is elapsed
 		countdown: false,			// If true, this will render the timer as a countdown (must have duration)
 		format: null,				// This sets the format in which the time will be printed
-		updateFrequency: 500		// How often should timer display update
+		updateFrequency: 500,		// How often should timer display update
+            	stopTimerOnDuration: true,	// If set to false than timer will keep on running even after duration is completed.
```

Comment:
Is this prop used anywhere? By the way, we do have a prop called `repeat` to do this. Hence we might not need this.

Reply:
removed the prop stopTimerOnDuration.